test/export/InstanceExporter.test.ts

```diff
@@ -11354,6 +11371,63 @@ describe('InstanceExporter', () => {
           ]
         });
       });
+
+      it('should assign an inline instance of a primitive to a primitive element', () => {
+        const primitiveInstannce = new Instance('MyFancyId')
```

Comment:
minor typo here and in following tests: variable name should be `primitiveInstance`

Reply:
Whoops, good call! Fixed in [5a78ac5](https://github.com/FHIR/sushi/pull/1539/commits/5a78ac52ec0a7a3fb931e8759d6c43a2e7a00bde).